meta-balena-common/classes/kernel-balena.bbclass

```diff
@@ -260,6 +261,11 @@ BALENA_CONFIGS[systemd] ?= " \
     CONFIG_FAIR_GROUP_SCHED=y \
     CONFIG_CFS_BANDWIDTH=y"
 
+BALENA_CONFIGS[optimize-size] ?= " \
+    CONFIG_CC_OPTIMIZE_FOR_SIZE=y \
+    CONFIG_SLOB=y \
+    "
+
```

Comment:
@jakogut could you paste some numbers for aarch64 & x86_64 kernel binaries?

Reply:
@alexgg x86_64:
```
--rw-r--r-- root       root         19404512 ./boot/bzImage
+-rw-r--r-- root       root         17136608 ./boot/bzImage
```

aarch64:
```
--rw-r--r-- root       root          9768338 ./boot/Image.gz-5.10.43-yocto-standard
+-rw-r--r-- root       root          8215595 ./boot/Image.gz-5.10.43-yocto-standard
```

Reply:
In addition to shrinking the kernel image, the optimize for size config also shrinks kernel modules.

Reply:
In addition to shrinking the kernel image, the optimize for size config also shrinks kernel modules.